### PR TITLE
Feb IDE release note

### PR DESCRIPTION
**Pull request by mirnawong1**
Feb IDE release note per request here: https://dbt-labs.slack.com/archives/C02NCQ9483C/p1677540936127859

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/docs/dbt-versions/release-notes/11-Feb-2023/feb-ide-updates.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+## New features 
+
+- Support for custom node colors in the IDE DAG visualization
+- Autosave prototype is now available under feature flag. [Contact](mailto:cloud-ide-feedback@dbtlabs.com) the dbt Labs IDE team to try this out
+- Ref autocomplete includes models from seeds and snapshots
+- Prevent menus from getting cropped (git controls dropdown, file tree dropdown, build button, editor tab options)
+- Additional option to access the file menu by right-clicking on the files and folders in the file tree
+- Rename files by double-clicking on files in the file tree and the editor tabs
+- Right-clicking on file tabs has new options and will now open at your cursor instead of in the middle of the tab
+- The git branch name above **Version Control** links to the repo for specific git providers
```
Comment (bossnunta) [inline website/docs/docs/dbt-versions/release-notes/11-Feb-2023/feb-ide-updates.md:22]: I would add a short note in parenthesis that this is technically only available for MT github and gitlab

File: website/docs/docs/dbt-versions/release-notes/11-Feb-2023/feb-ide-updates.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+- Implemented a new File Tree component to improve render time by 60%
+- Disabled the Local Storage of File Tree to prevent users from running into max LocalStorage issue for large projects
+- Changed snapshot snippet template (`__snapshot`) to a select from source
+
+## Bug fixes
+
+- You no longer have file contents carrying over when you switch to a different project that has the same file name
+- The preview max limit no longer allows you to override the maximum 
+- You no longer encounter node statuses failing to update in the history drawer for those on version 1.4 core. (This is a partial fix that may be fully addressed by core version 1.5)
+- You can now use the **Copy File Name** option to copy up to the first dot, rather than the final dot
```
Comment (bossnunta) [inline website/docs/docs/dbt-versions/release-notes/11-Feb-2023/feb-ide-updates.md]: The opposite actually --- copy to last dot, rather than first dot

**Review by bossnunta — DISMISSED** [review]
minor changes but otherwise looks pretty good!

File: website/docs/docs/dbt-versions/release-notes/11-Feb-2023/feb-ide-updates.md (hunk 1 above)
Comment (mirnawong1) [reply website/docs/docs/dbt-versions/release-notes/11-Feb-2023/feb-ide-updates.md:22]: gotcha thanks! to clarify, you mean US MT? or all MT instances (emea, us, apac)?

**Comment by mirnawong1** [comment]
thanks so much @bossnunta ! made those changes and requested a docs review. will close the loop when it's published!

File: website/docs/docs/dbt-versions/release-notes/11-Feb-2023/feb-ide-updates.md (hunk 1 above)
Comment (mirnawong1) [reply website/docs/docs/dbt-versions/release-notes/11-Feb-2023/feb-ide-updates.md:22]: stefan confirmed its all mt

Comment (bossnunta) [reply website/docs/docs/dbt-versions/release-notes/11-Feb-2023/feb-ide-updates.md:22]: all of MT yes!

**Review by matthewshaver — APPROVED** [review]

**Pull request merged**